src/components/ChatEntry.js

```diff
@@ -1,22 +1,53 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+  const onLikeButton = () => {
+    const updatedChat = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked
+    };
+    props.onUpdate(updatedChat)
+  };
+  const like = props.liked ? '❤️' : '🤍';
+
+  if (props.sender === 'Vladimir') {
+    return (
```

Comment:
Though, while this is a fine way to mark the remote sender, It does cause a lot of repetition just for the small change of the `className` attribute for the `div`.

You could avoid the repetition by doing something like:

```js
const divClassName = `chat-entry ${props.sender == 'Vladimir' ? 'local' : 'remote'}`; 
return (
    <div className={divClassName}>
{/* ...  rest of code ... */}
```